static/alphabetical_search.scss

```diff
@@ -239,4 +239,9 @@ padding: 8px 16px;
 
 .flex-end {
   justify-content: flex-end;
-}
\ No newline at end of file
+}
+
+.one-half-div {
+	float: left;
+	width: 50%;
+  }
```

Comment:
This should be at the start of the line?

Reply:
Sorry what do you mean here?

Reply:
All done now